src/Me/components/Input/Input.js

```diff
@@ -19,7 +19,7 @@ const StyledInput = styled.input`
   }
 `;
 
-export const Input = props => {
+export const Input = ({ value, ...props }) => {
   const id = useContext(formFieldContext);
-  return <StyledInput id={id} {...props} />;
+  return <StyledInput id={id} value={value ?? ''} {...props} />;
```

Comment:
Why is it needed?

Reply:
Why taking control of it but not letting the consumer to deal with it?

Reply:
Not sure I understand what you mean by that.
basically if you try to edit mentor profile right now you will see error from React. Becuase initially the value you pass to the input is `undefined` until you start typing. So you go from uncontrolled to controlled. React don't like it.

Reply:
So it basically covers the consumer fault. If the value shouldn't be `undefined` in the first place, let's make sure the consumer knows it.